vendor/k8s.io/code-generator/hack/boilerplate.go.txt

```diff
@@ -1,3 +1,16 @@
+Copyright 2019 Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
```

Comment:
file does not exist anymore

Reply:
removed it from the change adding license headers